tests/integration_tests/performance/test_network_ab.py

```diff
@@ -133,4 +136,16 @@ def test_network_tcp_throughput(
     )
     data = test.run_test(network_microvm.vcpus_count + 2)
 
+    cpu_load_path = Path(results_dir / "cpu_load.json")
+    with open(cpu_load_path, "w", encoding="utf-8") as f:
```

Comment:
nit(for all of these): `cpu_load_path.write_text(json.dumps(data["cpu_load_raw"])`

also, why split everything across so many files? This could just be a single json file into which all of `data` is dumped. That way if we ever change the structure of the `data` dict, we dont have to worry about also forgetting to update this. 

But actually, since we already updated the metrics thingy to always dump all emitted metrics into test_result, why do we need to put the same data into separate json files again in the first place?

Reply:
Will remove the cpu usage files for now as it indeed will copy what is in the metrics anyway.
Regarding other files, this is because I want to save all original iperf files we have, so we will have access to all data we currently do not emit as metrics.
Also in general I would prefer for us to remove all metric emitting from tests and do that as a separate post step which will look at the original data we saved and retrieve whatever metrics it needs. The metric retrieval can also be adopted by A/B framework later.